### ISSUE TITLE
[Design] Profile: Confirm proper styling of edit buttons and edit forms on mobile

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#26529]
## Background
As we've been working our way through correcting links that should be buttons in all areas of Profile, we need to determine answers to the following:

1. Should contact info and direct deposit info "edit" buttons be full width on mobile?
2. Should the buttons on the contact info and direct deposit edit forms be full width on mobile?

## Tasks
- [x] Determine answers to the two questions above
- [x] Provide any necessary design guidance for the FE devs to proceed with their work

**Comment by andaleliz** [comment]
@erikphansen I now see the problem - my mocks reflecting the updated to the secondary button style don't follow the design system guidance. That was my bad, sorry for the re-work.

We should follow the design system and let them go full width. 
Update and cancel buttons should stack.

Do you need me to provide mock-ups?  Thanks for catching this!

**Comment by erikphansen** [comment]
No mocks needed. I think it was this way because the "cancel" button used to look like a text link that sat right next to the update/save button.

**Comment by andaleliz** [comment]
That's what I thought too, but I went back to an old screenshot and see that indeed, the text link button style fell underneath the full-width update button.

FYI I noticed that the `back` and `continue` buttons in our forms aren't full width, so I [checked in with the design system team to learn more about exceptions to things being full width](https://dsva.slack.com/archives/C0NGDDXME/p1625592628213200?thread_ts=1625591989.213100&cid=C0NGDDXME) - I can see how it makes sense to have them side by side when you have a pair of buttons that perform opposite actions.  

I don't think full width is a usability problem or anything but just curious to learn more about any "rules" we have about this.  I'll report back if there are any notable updates/responses from them.
